src/__tests__/a11yAPI.test.js

```diff
@@ -9,9 +9,12 @@ const TEXT_LABEL = 'cool text';
 const TEXT_HINT = 'static text';
 // Little hack to make all the methods happy with type
 const NO_MATCHES_TEXT: any = 'not-existent-element';
-const NO_INSTANCES_FOUND = 'No instances found';
 const FOUND_TWO_INSTANCES = 'Expected 1 but found 2 instances';
 
+const getNoInstancesFoundMessage = (prop: string, value: string = NO_MATCHES_TEXT) => {
+  return `No instances found with ${name} '${value}'`;
```

Comment:
Let's use double quote:
```suggestion
  return `No instances found with ${name} "${value}"`;
```